connexion/middleware/exceptions.py

```diff
@@ -52,7 +52,7 @@ def http_exception(_request: StarletteRequest, exc: HTTPException) -> Response:
 
     @staticmethod
     def common_error_handler(_request: StarletteRequest, exc: Exception) -> Response:
-        logger.error(exc)
+        logger.error(exc, exc_info=exc)
```

Comment:
I guess this is equivalent to `logger.exception(exc)`?

Reply:
Not entirely, it would be the same as `logger.exception(exc, exc_info=exc)`

Source code:
```python
    def exception(self, msg, *args, exc_info=True, **kwargs):
        """
        Convenience method for logging an ERROR with exception information.
        """
        self.error(msg, *args, exc_info=exc_info, **kwargs)
```

But will not work properly as were not in an exception handler scope here